recipes/phasefieldx/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "phasefieldx" %}
+{% set version = "0.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/CastillonMiguel/phasefieldx/archive/v{{ version }}.tar.gz
+  sha256: f85113793ba1f199647f715a94ee5f36cc96f6288aef0dbe227812e8e7b70402
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip:
+    - win-64
+
+requirements:
+  host:
+    - python >=3.8
+    - hatchling
+    - pip
+  run:
+    - python >=3.8
+    - fenics-dolfinx ==0.8.0
+    - gmsh
+    - matplotlib
+    - mpi4py
+    - mpich
+    - pandas
+    - petsc4py
+    - numpy
+    - pyvista
+    - time
```

Comment:
I cannot see any of those dependencies in the metadata: https://github.com/CastillonMiguel/phasefieldx/blob/v0.0.3/pyproject.toml#L13
 
```suggestion
```

If they are rally needed, can you open an issue upstream?

Reply:
The dependencies are indeed necessary, and I’ve specified some of them in the installation instructions. I will make sure to include all of them

Reply:
https://github.com/CastillonMiguel/phasefieldx/commit/3395d76abde845fbbaab7a98b108ca8e05abde74